### DISCUSSION TITLE
libcurl 7.69 < 8.4.0 Heap Buffer Overflow

### DISCUSSION THREAD

**Discussion by machetto** [elfmz/far2l#2272] [General]
According to our sec system the latest FAR2L release has a vulnerability. Wondering if this could be fixed in the next release.

```
libcurl 7.69 < 8.4.0 Heap Buffer Overflow
The remote curl install is affected by a heap buffer overflow flaw.
Upgrade libcurl to version 8.4.0 or later
Path : /Applications/far2l.app/Contents/Frameworks/libcurl.4.dylib
Installed version : 7.88.1
Fixed version : 8.4.0
```

**Comment by elfmz** [comment]
Brief analyzis shown that only wxWidgets linked with that libcurl, however far2l doesnt use any networking functionality from wxWidgets, so that libcurl likely not used at all.

**Comment by elfmz** [comment]
Bundled libcurl (and many other libs) updated in recent release, and now its v8.8.0
